tests/suites/test_suite_net.function

```diff
@@ -26,3 +70,63 @@ void context_init_free( int reinit )
     goto exit;
 }
 /* END_CASE */
+
+/* BEGIN_CASE depends_on:MBEDTLS_PLATFORM_IS_UNIXLIKE */
+void poll_beyond_fd_setsize( )
+{
+    /* Test that mbedtls_net_poll does not misbehave when given a file
+     * descriptor beyond FD_SETSIZE. This code is specific to platforms
+     * with a Unix-like select() function. */
+
+    struct rlimit rlim_nofile;
+    int restore_rlim_nofile = 0;
+    int ret;
+    mbedtls_net_context ctx;
+    uint8_t buf[1];
+
+    mbedtls_net_init( &ctx );
+
+    /* On many systems, by default, the maximum permitted file descriptor
+     * number is less or equal to FD_SETSIZE. If so, raise the limit if
+     * possible.
+     *
+     * If the limit can't be raised, a newly open file descriptor
```

Comment:
I think `newly open` here refers to the previous strategy and should be updated.

Reply:
I was actually referring to the fact that the descriptor could have been opened before lowering the limit. The test wouldn't detect a bug whereby `FD_SETSIZE` is rejected at opening time but not at poll time. We know that this doesn't happen because we know how our code is written, but this could conceivably be an issue with some other platform's implementation of the `net_sockets` module — which is moot for testing purposes because the test wouldn't run on that platform anyway.

Reply:
Oh, I misunderstood, thanks for clarifying.